tabular/tests/unittests/models/test_automm.py

```diff
@@ -3,7 +3,6 @@
 
 @pytest.mark.gpu
 def test_automm_sts(fit_helper):
-    pytest.skip("Temporary skip the unittest")
```

Comment:
Why we skip it?

Reply:
We skipped this test previously because AutoMM was unstable in unit tests and we weren't actually using AutoMM in any Tabular config. It is no longer unstable and now with this switch we are directly using AutoMM in the multimodal tabular config, so we should enable the test. (Technically it should be have been enabled earlier, but wasn't)